src/TagSelector.xml

```diff
@@ -57,6 +57,10 @@
                 <caption>Enable Create</caption>
                 <description>Enable the Tag Selector to create tags when none are found.</description>
             </property>
+            <property key="additionalChars" type="boolean" defaultValue="false">
```

Comment:
Hmm is there no other property type that would allow for a bit more flexibility?   Or maybe have two separate booleans, one for comma one for space?

Reply:
Catalog specific requirement iis to add comma and space as of now , similar to sprintr functionality.
Do you thing enumeration options in the configuration(`comma , space , both` ) can make it more flexible ?

Reply:
added separate booleans for comma and space